graphql/schema/schema.graphql

```diff
@@ -72,30 +72,49 @@ type Query {
   listGalleryScrapers: [Scraper!]!
   listMovieScrapers: [Scraper!]!
 
-  """Scrape a list of performers based on name"""
-  scrapePerformerList(scraper_id: ID!, query: String!): [ScrapedPerformer!]!
-  """Scrapes a complete performer record based on a scrapePerformerList result"""
-  scrapePerformer(scraper_id: ID!, scraped_performer: ScrapedPerformerInput!): ScrapedPerformer
+  """Scrape for a single scene"""
+  scrapeSingleScene(source: ScraperSourceInput!, input: ScrapeSingleSceneInput!): [ScrapedScene!]!
+  """Scrape for multiple scenes"""
+  scrapeMultiScenes(source: ScraperSourceInput!, input: ScrapeMultiScenesInput!): [[ScrapedScene!]!]!
+
+  """Scrape for a single performer"""
+  scrapeSinglePerformer(source: ScraperSourceInput!, input: ScrapeSinglePerformerInput!): [ScrapedPerformer!]!
+  """Scrape for multiple performers"""
+  scrapeMultiPerformers(source: ScraperSourceInput!, input: ScrapeMultiPerformersInput!): [[ScrapedPerformer!]!]!
+
+  """Scrape for a single gallery"""
+  scrapeSingleGallery(source: ScraperSourceInput!, input: ScrapeSingleGalleryInput!): [ScrapedGallery!]!
+
+  """Scrape for a single movie"""
+  scrapeSingleMovie(source: ScraperSourceInput!, input: ScrapeSingleMovieInput!): [ScrapedMovie!]!
+
   """Scrapes a complete performer record based on a URL"""
   scrapePerformerURL(url: String!): ScrapedPerformer
-  """Scrapes a complete scene record based on an existing scene"""
-  scrapeScene(scraper_id: ID!, scene: SceneUpdateInput!): ScrapedScene
   """Scrapes a complete performer record based on a URL"""
   scrapeSceneURL(url: String!): ScrapedScene
-  """Scrapes a complete gallery record based on an existing gallery"""
-  scrapeGallery(scraper_id: ID!, gallery: GalleryUpdateInput!): ScrapedGallery
   """Scrapes a complete gallery record based on a URL"""
   scrapeGalleryURL(url: String!): ScrapedGallery
   """Scrapes a complete movie record based on a URL"""
   scrapeMovieURL(url: String!): ScrapedMovie
 
-  """Scrape a performer using Freeones"""
+  """Scrape a list of performers based on name (@deprecated - use scrapeSinglePerformer)"""
```

Comment:
When I used the `@deprecated` directive I found that it had to be used on the actual field declaration, not on the comment line. But it might be that doesn't work because it's the `Query` type? Not sure though.

Reply:
There's no official directive for deprecating operations as far as I can tell. I just used this for documentation.

Reply:
Just did test applying `@deprecated(reason: "...")` on an actual line, and it seems like it works just fine. As an executable build with `make generate ui build` runs without any issues.

And if I understand the specification correctly then `@deprecated` can (only) be applied on fields on a `type`. And `type Query { ... }` seems to be a type like any other. So I'm guessing the spec means you can't do `schema { query: Query @deprecated(reason: "...") }`

Reply:
Thanks, TIL.